packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -1081,6 +1099,7 @@ export default class SalesforceCRMService implements CRM {
           });
           if (extractedText) {
             writeOnRecordBody[field.name] = extractedText;
+            continue;
```

Comment:
Since each field has one type these continue statements are called after the field is processed by it's respective field type function.